Controller/ResourceController.php

```diff
@@ -39,11 +45,104 @@ public function __construct(
         $this->registry = $registry;
     }
 
-    public function resourceAction($repositoryName, $path)
+    public function getResourceAction($repositoryName, $path)
     {
         $repository = $this->registry->get($repositoryName);
         $resource = $repository->get('/'.$path);
 
+        return $this->createResponse($resource);
+    }
+
+    /**
+     * There are two different changes that can currently be done on a cmf resource:
```

Comment:
the styleci complains because the first line of a comment is the short description. 

we should say something like "Change an existing resource." and then add a new line.